edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -321,6 +324,68 @@ impl ModuleRuntime for DockerModuleRuntime {
         Box::new(result)
     }
 
+    fn get(&self, id: &str) -> Self::GetFuture {
+        fn parse_response<'de, D>(resp: &InlineResponse200) -> std::result::Result<String, D::Error>
+        where
+            D: serde::Deserializer<'de>,
+        {
+            let name = resp
+                .name()
+                .map(ToOwned::to_owned)
+                .ok_or_else(|| serde::de::Error::missing_field("Name"))?;
+            Ok(name)
+        }
+
+        debug!("Getting module {}...", id);
+
+        let id = id.to_string();
+
+        if let Err(err) = ensure_not_empty_with_context(&id, || {
+            ErrorKind::RuntimeOperation(RuntimeOperation::GetModule(id.clone()))
+        }) {
+            return Box::new(future::err(Error::from(err)));
+        }
+
+        let client_copy = self.client.clone();
+
+        Box::new(
+            self.client
+                .container_api()
+                .container_inspect(&id, false)
+                .then(|result| match result {
+                    Ok(container) => {
+                        let name = parse_response::<
+                            &mut serde_json::Deserializer<serde_json::de::IoRead<std::io::Empty>>,
+                        >(&container)
+                        .with_context(|_| {
+                            ErrorKind::RuntimeOperation(RuntimeOperation::GetModule(id.clone()))
+                        })?;
+                        let config =
+                            DockerConfig::new(name.clone(), ContainerCreateBody::new(), None)
```

Comment:
`edgelet/doc/modules.md` says, "The config is only used by the implementation of the corresponding ModuleRuntime trait. This means that the core of the daemon does not need to understand this config. It is an opaque blob of information that the implementation of the ModuleRuntime trait understands."

I didn't bother to populate the config object because it isn't intended to be used by callers of `get`, at least not currently.